olamilekan000/Politico
# Ft #163714446 user create account

#### What does this PR do?
Allows a user to create an account.

#### Description of Task to be completed?
- Setup JWT
- Use JWT to create a token
- Check if the user exists before saving into the Database
- If the user does not exist, then, the user is saved
- Respond with a the token and user that was created.

#### How should this be manually tested?
- Clone repo into your machine `git clone ft-user-create-account-163714446`
- Cd into the directory.
- Do `npm i` to install all dependencies.
- Open Postman
- Send a POST request to this url `localhost:8089/api/v1/auth/signup`
POST {
      firstname: 'Chigozie',
      lastname: 'Anidi',
      othername: 'Chigoze',
      email: 'chionline@gmail.com',
      phoneNumber: '08089562392',
      passportUrl: 'https://pbs.twimg.com/profile_images/852874627181367297/BhCGvY6d_200x200.jpg',
      isAdmin: true,
     password: 'secretlysecret',
}
- You should get a response with a token that looks similar to the object below
`
    Token:"jkjkjkjgkjrgk.jkikjrirugogoosdid.ofioifoeifoifeooechjh",
    user: {
      firstname: 'Chigozie',
      lastname: 'Anidi',
      othername: 'Chigoze',
      email: 'chionline@gmail.com',
      phoneNumber: '08089562392',
     passportUrl: 'https://pbs.twimg.com/profile_images/852874627181367297/BhCGvY6d_200x200.jpg',
     isAdmin: true
 }
`
#### What are the relevant pivotal tracker stories?
- A user can create an account
- https://www.pivotaltracker.com/story/show/163714446
